integrations/signup_test.go

```diff
@@ -5,6 +5,7 @@
 package integrations
 
 import (
+	"code.gitea.io/gitea/models"
```

Comment:
Sort imports

Reply:
Would it be worthwhile adding something like [gci](https://github.com/daixiang0/gci) to the ci workflow to enforce import ordering?

Reply:
it already does fail ;)

Reply:
Hmm, I'm confused then. It looks to me like [the CI build](https://drone.gitea.io/go-gitea/gitea/41663) succeeded?

Reply:
strange but true - perhaps "go vet" do not account testing files

Reply:
That does appear to be [the case](https://github.com/go-gitea/gitea/blob/27c15782352aabaf9279c9c187d3658649d81e2f/Makefile#L96):

```
GO_PACKAGES ?= $(filter-out code.gitea.io/gitea/models/migrations code.gitea.io/gitea/integrations/migration-test code.gitea.io/gitea/integrations,$(shell $(GO) list -mod=vendor ./... | grep -v /vendor/))
```